public/index.html

```diff
@@ -5,79 +5,108 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
 
+    <link rel="stylesheet" href="./css/fluent.css">
+    <link rel="stylesheet" href="./css/font-awesome.min.css">
     <link href="https://fonts.googleapis.com/css?family=Sarina" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
 
     <title>Project Fluent</title>
   </head>
   <body>
-    <div class="main">
-      <header class="header">
-        <h1 class="header__wordmark">
-          fluent
+    <div>
+      <nav id="topbar">
+        <div class="logo">
+          <a href="/">fluent</a>
+        </div>
+        <ul>
+          <li>
+            <a href="#">Home</a>
+          </li>
+          <li>
+            <a href="/fluent/guide">Documentation</a>
+          </li>
+          <li>
+            <a href="/play">Online Editor</a>
+          </li>
+          <li>
+            <a href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <div>
+      <header id="intro">
+        <h1>
+          Fluent
         </h1>
-        <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
-        </nav>
+        <h2>A localization paradigm designed to unleash the expressive power of the natural language.</h2>
+        <a href="https://github.com/projectfluent/fluent/wiki/Integrating-Fluent" class="cta">Get Started</a>
       </header>
-
-      <section class="section section--centered">
-        <p class="pitch">
-
-          <strong class="highlight">Fluent</strong> is a localization paradigm
-          designed to unleash the expressive power of the natural language.
-
+    </div>
+    <div>
+      <main>
+        <ul>
+          <li>
+            <h1>
+              Innovative
+            </h1>
+            <p>Natural-sounding translations that use the grammatical features of each language..</p>
+          </li>
+          <li>
+            <h1>
+              Powerfull & Extensible
+            </h1>
+            <p>
+              Date, time, number, plural rules support. Full bidirectionality.
+              Any internationalization formatters can be added.
```

Comment:
Perhaps `Custom formatters` would suffice? You don't use verb in other sentences. It would also be good to mention ICU, ECMA 402 and the Apache license here.